bin/node/service/src/parachain/service.rs

```diff
@@ -246,6 +253,10 @@ where
                 rpc_pool.clone(),
                 deny_unsafe,
             )));
+            io.extend_with(PubSubApi::to_delegate(PubSubApi::new(
+                pubsub.clone(),
+                Arc::new(Mutex::new(HashMap::new())),
```

Comment:
I believe this is part of module internals, no need to create it outside, just instance it inside of new method

Reply:
I mean hashmap of course